storm/src/main/storm/mesos/util/PrettyProtobuf.java

```diff
@@ -230,4 +230,19 @@ private static String setToString(Set set) {
     }
     return map;
   }
+
+  /**
+   * Wrapper around getTrimmedString which allows using gauva's transform utility.
+   */
+  private static Function<OfferID, String> offerIDToStringTransform =
+      new Function<OfferID, String>() {
+        public String apply(OfferID o) {
+          return o.getValue().toString();
+        }
+      };
+
+  public static String offerIDListToString(List<OfferID> offerIDList) {
+    List<String> offerIDsAsStrings = Lists.transform(offerIDList, offerIDToStringTransform);
+    return "[" + StringUtils.join(offerIDsAsStrings, ", ") + "]";
```

Comment:
Please try to follow this new non-+ strategy, like above:
`String.format("[%s]", StringUtils.join(rangesAsStrings, ","));`